File: go/batch_job_item_failure_response.go
// this file is @generated
package meteroid

type BatchJobItemFailureResponse struct {
	ChunkId BatchJobChunkId `json:"chunk_id"`

	Id string `json:"id"`

	ItemIdentifier *string `json:"item_identifier,omitempty"`

	ItemIndex int32 `json:"item_index"`

	Reason string `json:"reason"`
}

File: python/meteroid/models/batch_job_item_failure_response.py
# this file is @generated
import dataclasses
import typing as t

from ..serialization import BaseModel
from .batch_job_chunk_id import BatchJobChunkId


@dataclasses.dataclass
class BatchJobItemFailureResponse(BaseModel):
    chunk_id: BatchJobChunkId

    id: str

    item_index: int

    reason: str

    item_identifier: t.Optional[str] = None
